src/internal/connector/graph_connector_test.go

```diff
@@ -154,15 +154,16 @@ func (suite *GraphConnectorIntegrationSuite) TestExchangeDataCollection() {
 				return sel.Selector
 			},
 		},
-		{
-			name: suite.user + " Events",
-			getSelector: func(t *testing.T) selectors.Selector {
-				sel := selectors.NewExchangeBackup()
-				sel.Include(sel.EventCalendars([]string{suite.user}, []string{exchange.DefaultCalendar}, selectors.PrefixMatch()))
-
-				return sel.Selector
-			},
-		},
+		// {
```

Comment:
The changes to the test are to disable "Event restore" tests since we're debugging an issue with our test tenant calendars + Graph API.